core_impl/src/test/java/io/opencensus/implcore/stats/ViewManagerImplTest.java

```diff
@@ -270,7 +271,7 @@ public void testRecordInterval() {
 
```

Comment:
Should the two interval view tests also test with `DISTRIBUTION`, once sum of squared deviations is fully implemented?  I think there should be a TODO.

Reply:
Done.